src/libstd/rt/unwind/gcc.rs

```diff
@@ -129,7 +120,10 @@ pub mod eabi {
             uw::_URC_HANDLER_FOUND // catch!
         }
         else { // cleanup phase
-            uw::_URC_INSTALL_CONTEXT
+            unsafe {
+                __gcc_personality_v0(version, actions, exception_class, ue_header,
+                                     context)
+            }
```

Comment:
Oh hm ok I think I see what's going on here, it's because this is legit running in something other than the search phase, so we probably need to do something more than just returning a constant.